tests/lib/rules/jsx-indent.js

```diff
@@ -137,18 +157,23 @@ ruleTester.run('jsx-indent', rule, {
     ].join('\n'),
     options: [2],
     parserOptions: parserOptions,
-    errors: [{message: 'Expected indentation of 2 space characters but found 9.'}]
+    errors: [
+      {message: 'Expected indentation of 2 space characters but found 9.'},
+      {message: 'Expected indentation of 2 space characters but found 9.'}
```

Comment:
Why do you have 2 errors here? We should only have one in this case (for the closing tag).